src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -24,6 +24,9 @@ public partial class DllNotFoundException : System.TypeLoadException
         public DllNotFoundException(string message) { }
         public DllNotFoundException(string message, System.Exception inner) { }
     }
+    public partial struct RuntimeArgumentHandle
```

Comment:
Also, why are we adding this type? It is part of bundle of types that has to do with varargs (ArgIterator, etc.). If ArgIterator is excluded, this one should be excluded as well.